lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -60,8 +60,8 @@ class PendingWithoutReason < Base
         MSG = 'Give the reason for pending or skip.'
 
         # @!method pending_by_example_method?(node)
-        def_node_matcher :pending_by_example_method?, block_pattern(<<~PATTERN)
-          #Examples.pending
+        def_node_matcher :pending_by_example_method?, <<~PATTERN
```

Comment:
It may be that `block_node` has nothing to do with numblock.
https://github.com/rubocop/rubocop-rspec/blob/47f555b30894c75daab2e65394cc70e6f936d64e/lib/rubocop/rspec/language/node_pattern.rb#L12-L14

Reply:
It may be that `block_node` has nothing to do with numblock.
https://github.com/rubocop/rubocop-rspec/blob/47f555b30894c75daab2e65394cc70e6f936d64e/lib/rubocop/rspec/language/node_pattern.rb#L12-L14